Sources/SwiftWin32/Support/WindowMessage.swift

```diff
@@ -0,0 +1,1025 @@
+/**
+ * Copyright © 2021 Ryan Allan <rjallan@g.ucla.edu>
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ **/
+
+import WinSDK
+
+struct WindowMessage {
+  static func debugDescription(for uint: UINT) -> String {
+    if let message = messages[Int(uint)] {
+      return message
+    }
+    return "Unknown Message"
```

Comment:
This could be written more succinctly as:

```suggestion
    return messages[Int(uMsg)] ?? "Unknown Message"
```